src/nvim/option_defs.h

```diff
@@ -576,12 +576,13 @@ EXTERN char_u   *p_ttym;        /* 'ttymouse' */
 EXTERN unsigned ttym_flags;
 # ifdef IN_OPTION_C
 static char *(p_ttym_values[]) =
-{"xterm", "xterm2", "dec", "netterm", "urxvt", "sgr", NULL};
+{"xterm", "xterm2", "dec", "netterm", "pterm", "urxvt", "sgr", NULL};
```

Comment:
Sorry, this commit shouldn't have been in this PR, this is for #1881
Yes, reverting only this change fixes it